lib/licensed/sources/swift.rb

```diff
@@ -15,11 +15,12 @@ def enumerate_dependencies
         pins.map { |pin|
           name = pin["package"]
           version = pin.dig("state", "version")
+          url = pin["repositoryURL"].sub(/\.git$/, "")
```

Comment:
```suggestion
          url = pin["repositoryURL"]
          homepage = url.sub(/\.git$/, "")
```

Although not as common, Git URLs can be specified with non-HTTP schemes. A more robust solution would be to implement a `homepage_for_url` helper method that returns `nil` in those cases.